daml-assistant/integration-tests/src/Main.hs

```diff
@@ -279,6 +292,17 @@ packagingTests tmpDir = testGroup "packaging"
         callCommandQuiet $ unwords ["daml", "migrate", projectMigrate, "daml/Main.daml", aDar, bDar]
         step "Build migration project"
         withCurrentDirectory projectMigrate $ callCommandQuiet "daml build"
+        step "Merging upgrade dar"
+        withCurrentDirectory tmpDir $
+            callCommandQuiet $
+            unwords
+                [ "daml damlc merge-dars"
+                , projectA </> distDir </> "a.dar"
+                , projectB </> distDir </> "b.dar"
+                , "--dar-name"
+                , "b_upgraded.dar"
+                ]
+        assertBool "b_upgraded.dar was not created." =<< doesFileExist bUpgradedDar
```

Comment:
Should we check more than just the existence of this file? E.g., we could check that each filepath found in the original dars also exists in the merged dar.

Reply:
hmm, that's close to rewriting the command in the test. Let's wait and see what goes wrong when people start using it.